### PR TITLE
Update images on /desktop/organisations

### LINKED ISSUE #7257: Update the /desktop/organisations page with new images
Update [Ubuntu in organisations](https://ubuntu.com/desktop/organisations) page baed on the [copy doc](https://docs.google.com/document/d/1q3_fQB3PUGbREsbCC4FnmHrpgn9nIWtMwyhDC6CXhNE/edit)

- [x] banner new screen of map? Peter
- [x] pre-installed - use dell laptop

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Update images on /desktop/organisations

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/desktop/organisations
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the banner and laptop images are as recommended in the attached issue

## Issue / Card

Fixes #7257

### CODE REVIEW

File: static/sass/styles.scss [hunk 1]
```diff
@@ -314,9 +314,9 @@ summary {
 }
 
 .organisations-hero {
-  background-image: url("#{$assets-path}214648ba-_M8A2368-final.jpg");
+  background-image: url("#{$assets-path}32d95e39-hero-laptop-org.jpg?w=2000");
```
Comment (deadlight) [inline static/sass/styles.scss:317]: This seems a bit large for the use. About 1/2 the pixels still looks good to me

**Comment by deadlight** [comment]
@pmahnke I think that the image size is a bit large and could be shrunk quite a bit to reduce the page weight (I get a noticeably delayed load without cache)

File: static/sass/styles.scss (hunk 1 above)
Comment (deadlight) [inline static/sass/styles.scss:317]:
```suggestion
  background-image: url("#{$assets-path}32d95e39-hero-laptop-org.jpg?w=1000");
```

Comment (pmahnke) [reply static/sass/styles.scss]: ok, I already dropped it a bunch.

**Pull request merged**